modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -496,15 +502,19 @@ Available users can be seen using `SHOW USERS`, which will produce a table of us
 | {check-mark}
 
 | roles
-| Roles granted to the user.
+| Native roles granted to the user using the `GRANT ROLE` command.
+
+This is not necessarily the set of roles that a user will receive in practice. The latter depends on DMBS configuration as well as the user's xref:authentication-authorization/auth-providers.adoc[Auth Providers].
+
+Examples of where a user would receive a different set of roles from that which appears in this column would include cases where they use external (e.g. LDAP or OIDC) auth, or where they do not have the `native` xref:authentication-authorization/auth-providers.adoc[Auth Provider].
```

Comment:
I think it's more if native isn't defined in config rather than if they have the object: as per previous discussion I believe you can have a user without a native auth object but still uses native roles for the authorization

Reply:
thanks. revised.